ocaml/xapi/helpers.ml

```diff
@@ -1674,6 +1674,27 @@ let try_internal_async ~__context (marshaller : Rpc.t -> 'b)
           info "try_internal_async: destroying task: t = ( %s )" ref ;
           TaskHelper.destroy ~__context t)
 
+module StunnelClient : sig
+  val get_verify_by_default : unit -> bool
+
+  val set_verify_by_default : bool -> unit
+end = struct
```

Comment:
Instead of modifying the local database when enabling/disabling verification, like on the feature/REQ-403 branch, we just use this module instead

Reply:
When enabling the verification for the whole pool this is called *in addition* to setting the database field.

Reply:
That is correct